lucien-ios/AddAndEditNewComicBook/ComicFormViewModel.swift

```diff
@@ -64,51 +64,42 @@ class ComicFormViewModel {
 
     // MARK: - Variables
 
-    var comicFormMode: ComicFormMode
-
+    var comicFormMode = ComicFormMode.add
+    var coverPhoto: UIImage?
     var navigationBarTitle: String {
         return comicFormMode == .add ? "Add Book" : "Edit Book"
     }
+
     var genreTitles: Observable<[String]> {
         return Observable.just(Comic.Genre.allCases.map { $0.title })
     }
+
     var conditionTitles: Observable<[String]> {
         return Observable.just(Comic.Condition.allCases.map { $0.title })
     }
 
-    var coverPhoto: UIImage?
-
     // MARK: - Private Variables
 
     private let disposeBag = DisposeBag()
     private let lucienAPIClient = LucienAPIClient()
+    private var comicID: String? 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Trailing Whitespace Violation: Lines should not have trailing whitespace. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.intrepid.io/coding_rules#rule_key=SwiftLint%3Atrailing_whitespace)